src/docs/release-notes/R10/FullNotes.mdx

```diff
@@ -20,7 +20,7 @@ We've expanded on our previous offering of a PX Blue icon font. We now have icon
 
 -   [Icon Font](https://www.npmjs.com/package/@pxblue/icons): All icons packaged into a single font
 -   [SVG Icons](https://www.npmjs.com/package/@pxblue/icons-svg): icons available as standalone SVGs (including a sprite sheet to be used with Angular applications)
--   [MUI Icons](https://www.npmjs.com/package/@pxblue/icons-mui): Material-UI-style icon components for use in React applications
+-   [MUI Icons](https://www.npmjs.com/package/@pxblue/icons-mui): MUI-style icon components for use in React applications
```

Comment:
This should NOT be changed here. It was still called Material-UI back then.

Reply:
wow in R10, not sure how that changed. ok

Reply:
@emclaug2 are you talking about the changes here?